test/AuxiliaryTest.cpp

```diff
@@ -54,20 +54,31 @@ TEST_CASE( "string_test", "[auxiliary]" )
     REQUIRE(expected3 == split("/path/to/relevant/data/", "/"));
 }
 
+namespace openPMD
+{
+namespace Test
+{
+struct S : public Writable
```

Comment:
maybe intend by one level here.

Reply:
Indenting the ```struct``` would make this one class inconsistent with all the rest of the API. I'm for leaving this one unchanged.

Reply:
ok I see, then we can make such changes later on.
Here is how we usually style nested namespaces: https://github.com/ComputationalRadiationPhysics/contributing/blob/master/codingGuideLines/cpp.md#6-namespaces

Reply:
I do not dislike the style, but that will require a change to every single C++ source file in this project.